test/dependencies/graphsync-get/graphsync-get.go

```diff
@@ -34,13 +34,13 @@ func newGraphsync(ctx context.Context, p2p host.Host, bs blockstore.Blockstore)
 	network := network.NewFromLibp2pHost(p2p)
 	return gsimpl.New(ctx,
 		network,
-		storeutil.LoaderForBlockstore(bs),
-		storeutil.StorerForBlockstore(bs),
+		storeutil.LinkSystemForBlockstore(bs),
+		//TODO: not needed anymore? // storeutil.StorerForBlockstore(bs),
```

Comment:
Yep this is fine, we did the same thing here https://github.com/ipfs/go-ipfs/commit/3f7e23e640d166e08e87d285fd53e98a1542ad6d